frontend/components/TableContainer/DataTable/DataTable.tsx

```diff
@@ -79,6 +79,7 @@ const DataTable = ({
         }, [sortHeader, sortDirection]),
       },
       disableMultiSort: true,
+      disableSortRemove: true,
```

Comment:
For clarity, let's rename this because it's not clear on what's being disabled.

Reply:
Lol, yeah I agree but that's what the prop is called in the [react-table docs](https://react-table.tanstack.com/docs/api/useSortBy). Very opaque one-liner. It took me forever scrubbing through the docs to figure out this was the prop I needed to change.

Reply:
Oh gosh. Well nevermind then lol. Thanks!